geoportailv3/models.py

```diff
@@ -77,3 +77,13 @@ class LuxGetfeatureDefinition(Base):
     attributes_to_remove = Column(Unicode, label=_(u'Attributes to keep'))
     poi_id_collection = Column(Unicode, label=_(u'Id of the poi collection'))
     geometry_column = Column(Unicode, label=_(u'Geometry column name'))
+
+
+class LuxPrintJob(Base):
+    __tablename__ = 'lux_print_job'
+    __table_args__ = {'schema': _schema}
+
+    id = Column(Integer, primary_key=True)
+    ref = Column(String, index=True, unique=True, nullable=False)
```

Comment:
Why not use `ref` as the primary key? I don't think we need a database sequence in this case.